core/arch/arm/plat-hikey/spi_test.c

```diff
@@ -33,10 +33,18 @@
 #include <trace.h>
 #include <util.h>
 
-#define PL022_STAT		0x00C
+#define PL022_STAT	0x00C
 #define PL022_STAT_BSY	SHIFT_U32(1, 4)
 
-static __maybe_unused void spi_cs_callback(enum gpio_level value)
+static void spi_some_delay(size_t mult)
+{
+	size_t i;
+
+	for (i = 0; i < (mult*10000000); i++)
+		;
+}
```

Comment:
I will `tee_time_mdelay` in a separate commit by itself below. Please check if it's ok. Thanks!